helm/templates/tests/test-connection.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: "{{ include "substrate-telemetry.fullname" . }}-test-connection"
+  labels:
+    {{- include "substrate-telemetry.labels" . | nindent 4 }}
+  annotations:
+    "helm.sh/hook": test
+spec:
+  containers:
+    - name: wget
+      image: busybox
+      command: ['wget']
+      args: ['{{ include "substrate-telemetry.fullname" . }}:{{ .Values.service.port }}']
```

Comment:
Should `/health` be appended to this (`/` will 404)?

Reply:
this was the default test in the helm chart, I removed it as we don't use this test. we use Kubernetes for liveness and readiness checks.